### PR TITLE
Fix old statements about implemented lookup tables

### PR DESCRIPTION
**Pull request by dannywillems**

### CODE REVIEW

File: book/src/kimchi/lookup.md [hunk 1]
```diff
@@ -125,7 +126,10 @@ $$
 
 Since we would like to allow multiple table lookups per row, we define multiple **queries**, where each query is associated with a **lookup selector**.
 
-At the moment of this writing, the `ChaCha` gates all perform $4$ queries in a row. Thus, $4$ is trivially the largest number of queries that happen in a row.
+Previously, ChaCha20 was implemented in Kimchi but has been removed as unneeded.
```
Comment (rbonichon) [inline book/src/kimchi/lookup.md:129]: suggestion: s/as unneeded/as it has become unneeded.

**Review by rbonichon — APPROVED** [review]
LGTM. I only have a single minor suggestion.

File: book/src/kimchi/lookup.md (hunk 1 above)
Comment (dannywillems) [inline book/src/kimchi/lookup.md:129]:
```suggestion
Previously, ChaCha20 was implemented in Kimchi but has been removed as it has become unneeded.
```

Comment (dannywillems) [reply book/src/kimchi/lookup.md]: Fixed.

**Pull request merged**